### DISCUSSION TITLE
more notifications please

### DISCUSSION THREAD

**Discussion by peterus** [community/community#3185] [Mobile] [answered]
currently i have some activity going on on some repos. and i need to check daily my mails for some new comments on merge requests and tickets. but i would like to have some notifications on my phone (via the github app (ios))! did i missed something im my settings? or can you add more notifications? thanks!

**Comment by candyho** [comment, marked as answer]
Hey thanks for your feedback! We're solely supporting **direct-mention** push notification at this point, and you can enable that through app Settings. Supporting more push notification types is definitely something on our radar. Hopefully we'll have some updates to share with you soon. Stay tuned!

**Reply by ntilley905** [reply]
Would also love to see more notifications! It is a bit confusing to have notifications on the website and also notifications on the Mobile App, yet they don't match.

**Reply by candyho** [reply]
Hey, do you mind letting us know what are the notifications you are missing on mobile specifically? Also please make sure you have checked the "Web and Mobile" box on the notifications settings page - https://github.com/settings/notifications
<img width="605" alt="Screenshot 2021-02-02 at 18 12 36" src="https://user-images.githubusercontent.com/7498102/106636473-430cd380-6582-11eb-85da-0f957542e3df.png">

**Reply by ntilley905** [reply]
Hi! Thanks for looking into this. I should've been more specific, I was just voicing my support for the additional push notifications as mentioned above. I do see all the notifications on my app as I do on the website, but it would be great to get those via push notifications to simplify my overcrowded email inbox.

**Reply by candyho** [reply]
Ah, gotcha! Yes adding more push types is something we are actively looking into, stay tuned. :)
